src/fonts.php

```diff
@@ -89,7 +89,9 @@ public static function enqueue_google_fonts( $google_fonts, $handle = 'stackable
 			}
 
 			foreach ( $google_fonts as &$font ) {
-				$font = str_replace( ' ', '+', $font ) . ':100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic';
+				if (!empty($font)) {
```

Comment:
```suggestion
				if ( ! empty( $font ) ) {
```

Reply:
Thanks for your suggestion and commit :)